src/numJewelsInStones.js

```diff
@@ -25,8 +25,19 @@
  *
  * @return {number}
  */
+
 function numJewelsInStones(jewels, stones) {
-  // write code here
+  let countJewels = 0;
```

Comment:
countJewels звучит как функция (посчитатьКамни)
```suggestion
  let counter = 0;
```